images/build/debian-base/bookworm/Dockerfile.build

```diff
@@ -41,24 +41,7 @@ RUN apt-get update \
     && apt-get dist-upgrade -y
 
 # Hold required packages to avoid breaking the installation of packages
-RUN apt-mark hold apt gnupg adduser passwd libsemanage1 libcap2
-
-# Remove unnecessary packages.
```

Comment:
Is there any reason to drop these with the inbound changes?

Reply:
I found no unnecessary packages, the list returned by the commands in the comment was empty. :shrug:

Reply:
The comment is perhaps misleading, the commands need a little more work — `apt-cache rdepends` needs to be run on each installed package individually to find the roots. At least `e2fsprogs` and its dependencies can still be removed, but it has to be done differently:

```sh
dpkg --purge e2fsprogs
apt-get autoremove --purge -y
```

(this takes care of `e2fsprogs`, `libss2`, `libcom-err2`, `libext2fs2`, and `logsave`).

Reply:
The `ncurses` packages can also be removed:

```sh
dpkg --purge --force-remove-essential ncurses-base ncurses-bin
```

`tzdata` too:

```sh
dpkg --purge tzdata
```

If `bash` is still not needed, it can be removed in the same way as the `ncurses` packages above. The remaining packages are dependencies of `util-linux`; I suspect it could be removed too but I’m not familiar enough with all the requirements to judge that.

All told, the result would end up looking like

```
RUN dpkg --purge --force-remove-essential bash e2fsprogs libss2 libcom-err2 libext2fs2 logsave ncurses-base ncurses-bin tzdata
```

Reply:
Makes sense, thank you for the clarification!